api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -283,6 +301,30 @@ public void saveProvider_shouldSaveAProviderWithPersonAlone() {
 		assertEquals(999, provider.getPerson().getId().intValue());
 		
 	}
+
+	/**
+	 * @see ProviderService#saveProvider(Provider)
+	 */
+	@Test
+	public void saveProvider_shouldSaveAProviderWithProviderRole() {
+		Provider provider = new Provider();
```

Comment:
Is this how existing tests confirm that an entity was persisted to the database?